components/chef-ui-library/src/charts/chef-radial-chart/chef-radial-chart.tsx

```diff
@@ -86,6 +86,11 @@ export class ChefRadialChart {
    */
   @Prop() id: string;
 
+  /**
+   * The width of the gap to apply between chart segments.
+   */
+  @Prop({ reflectToAttr: true }) gapSize = .03;
```

Comment:
I would like to see this converted to degrees instead of radians for two reasons:
1. Probably less folks think in radians than in degrees.
2. Much preferred to be able to specify gapSize as an integer instead of floating point.

Keep the public API as degrees; keep the conversion to radians internal to this file.